kf_lib_data_ingest/etl/load/load_v1.py

```diff
@@ -289,15 +292,15 @@ def _load_entity(self, entity_class, record):
 
         self.seen_entities[entity_class.class_name].add(unique_key)
 
+        target_id = self._get_target_id_from_record(entity_class, record)
+        method = "UPDATE" if target_id else "ADD"
```

Comment:
I know a large fraction of this PR refactors the same processes, but why don't we align these terms with either CRUD or request methods? For example:

- `"UPDATE"` >> either `"UPDATE"` (says same) or `"PATCH"` 
- `"ADD"` >> either `"CREATE"` or `"POST"`

Reply:
I didn't want the library to assume which HTTP verbs are used

Reply:
Nice. Let's do that then! Once changed, I will approve.